src/expressions/functions/builtInFunctions_numeric.ts

```diff
@@ -303,86 +303,113 @@ const fnNumber: FunctionDefinitionType = (
 	});
 };
 
-const permute: FunctionDefinitionType = (
-	_dynamicContext,
-	_executionParameters,
-	_staticContext,
-	sequence
-) => {
-	if (sequence.isEmpty() || sequence.isSingleton()) {
-		return sequence;
-	}
-
-	const a = sequence.getAllValues();
-
-	// Use Durstenfeld shuffle to randomize the list
-	for (let i = a.length - 1; i > 1; i--) {
-		const j = Math.floor(Math.random() * (i + 1));
-		const t = a[j];
-		a[j] = a[i];
-		a[i] = t;
+function createSeedFromString (seed: string): number {
+	let iseed = 0;
+	for (let i = 0; i < seed.length; ++i) {
+		iseed += seed.charCodeAt(i);
```

Comment:
Probably ok, but this "hash" might be a bit primitive - order of the characters in the string does not affect the hash and this always favors lower integers for smaller strings. Probably lots of other collisions there. No idea how that affects the randomness...

Reply:
I added a better hashing algorithm here